sensing/imu_corrector/schema/imu_corrector.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for imu_corrector",
+  "type": "object",
+  "definitions": {
+    "velodyne_monitor": {
+      "type": "object",
+      "properties": {
+        "angular_velocity_offset_x": {
+          "type": "number",
+          "description": "roll rate offset in imu_link [rad/s]."
+        },
+        "angular_velocity_offset_y": {
+          "type": "number",
+          "description": "itch rate offset imu_link [rad/s]."
```

Comment:
typo
```suggestion
          "description": "pitch rate offset imu_link [rad/s]."
```

Reply:
Thank you for suggestion..I updated the description  properly..